app/views/bulk_updates/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="row row-content">
+  <section>
+    <h2>Bulk Update <%= @task_id %></h2>
+
+    <div class="row">
+      <label class="bulk-update-status"><%= @task.fetch('task-status') %></label>
```

Comment:
KeyNotFound Error. Since we're here and need to make an update, both of these should probably just use `['']` given that they are defaulting to nil. `status-message` is also used below in this file, and could/should also use the `['']` because it also defaults to nil (if the key doesnt exist when using `['']` it just returns nil, so its the same result with less work)